### ISSUE TITLE
when merging info sets, use move names of first, not of last, merged info set

### ISSUE THREAD

**Issue by stengel** [gambitproject/jsgte#79]
When "merge info sets" (the link sign) appears, separate info sets have automatic separate move names:
![s6](https://cloud.githubusercontent.com/assets/798638/14235162/ef1091d6-f9ed-11e5-8c54-b7c71d6fda65.png)
When these three information sets are merged (I used multiaction, not sure the same happens when info sets are merged by first clicking one and then the other), the move names of the **last** merged info set are used, which now wastes the previous automatically chosen letters a,b,c,d:
![s7](https://cloud.githubusercontent.com/assets/798638/14235170/2b462468-f9ee-11e5-9a8d-751de02a19d5.png)

It is better to use the move names of the **first** merged info set, here a,b

**Comment by PARAG00991** [comment]
Yes, @stengel same thing is happening even in case, when info sets are merged by first clicking one info set and then the other.
